docs/composition.ipynb

```diff
@@ -0,0 +1,739 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can execute multiple queries at once using the `select` or `agg` methods. To start off, let's calculate the actual and differentially private mean and median for the variable `HWUSUAL`, which is the number of hours per week usually worked in the main job by respondents that were working for pay at the time of the survey.\n",
+    "\n",
+    "Notes: \n",
+    "- Remember to reference the [EU Labour Force Survey User Guide](https://ec.europa.eu/eurostat/documents/1978984/6037342/EULFS-Database-UserGuide.pdf) for any questions about variables. \n",
+    "- Do not calculate the actual mean to impute null values. We shouldn't reference our data after creating the compositor. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/dg/nlwnz_jj7rsblr2tzwj962b40000gn/T/ipykernel_41502/693093094.py:3: DeprecationWarning: `groupby` is deprecated. It has been renamed to `group_by`.\n",
+      "  actual_hours = df.groupby(\"ILOSTAT\").agg(\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (1, 5)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>ILOSTAT</th><th>Mean Hours Worked</th><th>Median Hours Worked</th><th>Mean Hours Worked_right</th><th>Median Hours Worked_right</th></tr><tr><td>i64</td><td>f64</td><td>f64</td><td>f64</td><td>i64</td></tr></thead><tbody><tr><td>1</td><td>37.667898</td><td>37.0</td><td>36.397198</td><td>25</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (1, 5)\n",
+       "┌─────────┬───────────────────┬─────────────────────┬──────────────┬───────────────────────────┐\n",
+       "│ ILOSTAT ┆ Mean Hours Worked ┆ Median Hours Worked ┆ Mean Hours   ┆ Median Hours Worked_right │\n",
+       "│ ---     ┆ ---               ┆ ---                 ┆ Worked_right ┆ ---                       │\n",
+       "│ i64     ┆ f64               ┆ f64                 ┆ ---          ┆ i64                       │\n",
+       "│         ┆                   ┆                     ┆ f64          ┆                           │\n",
+       "╞═════════╪═══════════════════╪═════════════════════╪══════════════╪═══════════════════════════╡\n",
+       "│ 1       ┆ 37.667898         ┆ 37.0                ┆ 36.397198    ┆ 25                        │\n",
+       "└─────────┴───────────────────┴─────────────────────┴──────────────┴───────────────────────────┘"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "candidates = list(range(25,56))\n",
```

Comment:
Maybe not on this page, but where we first introduce candidates for quantiles, do we explain that they don't need to be every integer in a range, and that that could be a better tradeoff in some cases?